charts/apps/values.yaml

```diff
@@ -10,3 +10,27 @@ bundler:
   repoUrl: https://github.com/DiamondLightSource/authz.git
   targetRevision: HEAD
   path: charts/bundler
+
+opa:
+  enabled: true
+  repoUrl: https://github.com/DiamondLightSource/authz.git
+  targetRevision: HEAD
+  path: charts/opa
+  valuesObject:
+    orgData:
+      bundlerSecret:
+        name: token-authorization
+        key: bearer
+    orgPolicy:
+      enabled: true
+      userinfoEndpoint: https://auth.diamond.ac.uk/cas/oidc/oidcProfile
```

Comment:
Is CAS still supported? Do we need OPA to have access to `JWKS_ENDPOINT` after #187?

Reply:
Good catch. CAS has been deprecated, will add the `JWKS_ENDPOINT` url. Do you have thoughts on which realm to point to?

Reply:
Have opted for `master`